src/map.cpp

```diff
@@ -805,7 +805,7 @@ bool mapLoad(char *filename, bool preview)
 	//load in the map data itself
 
 	/* Load in the map data */
-	for (i = 0; i < mapWidth * mapHeight; i++)
+	for (int i = 0; i < mapWidth * mapHeight; ++i)
```

Comment:
Should this be `unsigned` for consistency?

Reply:
If unsigned, it would give a warning about comparing unsigned int with signed int, since mapWidth and mapHeight are signed.

Reply:
Ah, didn't realise they were signed. Ok!